ref-exchange/src/rated_swap/README.md

```diff
@@ -0,0 +1,29 @@
+### RatedSwapPool for stNEAR/wNEAR pool
+
+Based on StableSwapPool source code
+
+Major changes:
+- added rates to swap & add/predict/remove_liquidity calculations
+- math upgraded to 24 decimals using U384
+- rates acquired from another smart contract via cross-contract call
+
+New external methods:
+- add_rated_swap_pool
+- add_rated_liquidity
+- rated_swap_ramp_amp
+- rated_swap_stop_ramp_amp
+- get_rated_pool [view]
+- predict_add_rated_liquidity [view]
```

Comment:
Besides adding liquidity, we also need an option parameter with token rate in predict_remove_liquidity_by_token, predict_swap.